docs/docs/10-intro.md

```diff
@@ -2,16 +2,16 @@
 
 Woodpecker is a simple CI engine with great extensibility.
 
-![woodpecker](/images/woodpecker.png)
+![woodpecker](woodpecker.png)
 
-## .drone.yml
+## .woodpecker.yml
```

Comment:
Is this really the case? When did this change? see: https://github.com/woodpecker-ci/woodpecker/pull/154

Reply:
I think we support both - in the log term I would say we should use `.woodpecker.yml` as the two projects can devide but use `.drone.yml` as fallback - if nothing is set in settings (settings overwrite it in all cases) ...

Reply:
Changing to some woodpecker related name is planned for 0.15.0 #133, but maybe it was a bit to quick to update it here already.

Reply:
no leave it ... but we have to resolve #133 first then :)